reggie/ingestion/preprocessor/colorado_preprocessor.py

```diff
@@ -8,6 +8,7 @@
 from datetime import datetime
 import gc
 from reggie.ingestion.utils import date_from_str, MissingNumColumnsError
+import chardet
```

Comment:
I think you will need to add this to reggie's requirements.txt and setup.py files. (I know it looks like Inspector already has it installed, but if people are running standalone reggie, they'll need it included there explicitly too)